byzcoin/service.go

```diff
@@ -1875,6 +1875,7 @@ func (s *Service) processOneTx(sst *stagingStateTrie, tx ClientTransaction) (Sta
 				}
 				return nil, nil, fmt.Errorf("%s: contract %s %s", s.ServerIdentity(), contractID, reason)
 			}
+			log.Lvlf2("StateChange %s for id %x - contract: %s", sc.StateAction, sc.InstanceID, sc.ContractID)
```

Comment:
Are you sure level 2 is correct here? Seems like a 3 thing?

Reply:
I would say that until we have a lot of transactions, it's really a useful debugging output. If we start having multi-transaction blocks regularly, we can change it to `Lvl3`. What do you think?

Reply:
ok, makes sense.